disperser/common/v2/blobstore/dynamo_metadata_store.go

```diff
@@ -854,6 +854,78 @@ func (s *BlobMetadataStore) GetDispersalRequest(ctx context.Context, batchHeader
 	return req, nil
 }
 
+// GetDispersalRequestByDispersedAt returns DispersalRequest within time range (start, end)
+// (both exclusive), retrieved and ordered by DispersedAt timestamp in specified order.
+//
+// If specified order is ascending (`ascending` is true), retrieve data from the oldest (`start`)
+// to the newest (`end`); otherwise retrieve by the opposite direction.
+//
+// If limit > 0, returns at most that many attestations. If limit <= 0, returns all results
+// in the time range.
+func (s *BlobMetadataStore) GetDispersalRequestByDispersedAt(
+	ctx context.Context,
+	operatorId core.OperatorID,
+	start uint64,
+	end uint64,
+	limit int,
+	ascending bool,
+) ([]*corev2.DispersalRequest, error) {
+	if start+1 > end-1 {
```

Comment:
If `start=1` and `end=2` the current conditional returns error (ie `2 > 1`) whereas `start >= end-1` will allow for a single time point (ie `1 >= 1`).